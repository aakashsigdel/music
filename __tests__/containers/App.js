import 'react-native'
import React from 'react'
import App from '../../src/containers'

import renderer from 'react-test-renderer'

it('should render <App /> correctly', () => {
  const tree = renderer.create(
    <App />
  ).toJSON()

  expect(tree).toMatchSnapshot()
})
