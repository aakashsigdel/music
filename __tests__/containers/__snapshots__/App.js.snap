exports[`test should render <App /> correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#70A1D7",
      "flex": 1,
    }
  } />
`;
